lib/file/index.js

```diff
@@ -73,12 +71,9 @@ File.prototype._validate = function() {
 File.prototype.addBatch = function(batch) {
 	
 	// Set the batch number on the header and control records
-	batch.header.batchNumber.value = batchSequenceNumber
-	batch.control.batchNumber.value = batchSequenceNumber
+	batch.header.batchNumber.value = this._batches.length
```

Comment:
Although this generates a sequential batch number, it could be useful to have a `batchSequenceNumber` that could be initialized when creating the `File`, instead of starting always with `0`. 

WDYGT?